### PR TITLE
Fix punc

### PR DESCRIPTION
**Pull request by wi-ski**

### CODE REVIEW

**Comment by erikd** [comment]
Please squash those commits down into a single commit so the addition/deletion of the `ff-genesis.json` will disappear.

File: doc/shelley-genesis.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ should be created offline, kept offline and only the verification keys shared.
 
 For a demo we're going to put all the keys together in once place
 
-To start with we will set up our template directory:
+To start, with we will set up our template directory:
```
Comment (mrBliss) [inline doc/shelley-genesis.md:60]: This doesn't improve things, IMO, "with we will set up ..."  is just wrong.

I'm not convinced this needs changing, but I would go for something like "We will start with setting up ...".

Comment (intricate) [reply doc/shelley-genesis.md:60]: or this could be formatted as:

```suggestion
To start with, we will set up our template directory:
```

**Comment by intricate** [comment]
Superseded by #1770

**Pull request closed without merging**